### PR TITLE
Add multiplier unit converter

### PR DESCRIPTION
**Pull request by syzygy333**
### Summary
Adds multiplier variable that looks at the units being used and adjusts itself accordingly.
### Risk
Medium

### CODE REVIEW

File: base/mixins/_fluid-units.scss [hunk 1]
```diff
@@ -54,10 +54,20 @@
     }
   }
 
+  // Check the unit and adjust the multiplier accordingly so this can be applied to more than just type
+  $multiplier: 1;
```
Comment (jhogue) [inline base/mixins/_fluid-units.scss:58]: Nitpick… `so this can be applied to more than just one unit of measurement`

**Comment by BrandonDH** [comment]
Lgtm @syzygy333 just J's comment and should be good to go

**Comment by BrandonDH** [comment]
@syzygy333 Anything else need to be done here?

**Comment by BrandonDH** [comment]
@syzygy333 You have anymore changes here?

**Pull request merged**